geode-core/src/test/java/org/apache/geode/test/dunit/standalone/DUnitLauncher.java

```diff
@@ -297,6 +297,8 @@ public Object call() throws IOException {
         // able to do so successfully anyway
         p.setProperty(DISABLE_AUTO_RECONNECT, "true");
 
+        System.setProperty("geode.feature-protobuf-protocol", "true");
```

Comment:
Ideally we would not set this all the time. However, it would be a non-trivial effort to only set this when we need it and the feature flag should go away soon entirely.